src/functional/fencil_processors/runners/roundtrip.py

```diff
@@ -118,12 +136,19 @@ def register_tmp(tmp, domain):
         ir.pre_walk_values().if_isinstance(AxisLiteral).getattr("value").to_set()
     )
 
-    header = """
-import numpy as np
-from functional.iterator.builtins import *
-from functional.iterator.runtime import *
-from functional.iterator.embedded import np_as_located_field
-"""
+    if use_embedded:
+        builtins_import = "from functional.iterator.embedded import *"
+    else:
+        builtins_import = "from functional.iterator.builtins import *"
+
```

Comment:
add comment

Reply:
Added a docstring to the method containing that information.